nixos/modules/config/fonts/fontconfig.nix

```diff
@@ -179,15 +170,21 @@ let cfg = config.fonts.fontconfig;
       ln -s ${renderConf}       $support_folder/conf.d/10-nixos-rendering.conf
       ln -s ${renderConf}       $latest_folder/conf.d/10-nixos-rendering.conf
 
-      # 30-nixos-generic-alias.conf
-      ln -s ${genericAliasConf} $support_folder/conf.d/30-nixos-generic-alias.conf
-      ln -s ${genericAliasConf} $latest_folder/conf.d/30-nixos-generic-alias.conf
+      # 50-user.conf
+      ${optionalString (! cfg.includeUserConf) ''
+      rm    $support_folder/conf.d/50-user.conf
+      rm    $latest_folder/conf.d/50-user.conf
+      ''}
 
-      # 99-user.conf
-      ${optionalString cfg.includeUserConf ''
-      ln -s ${userConf}         $support_folder/conf.d/99-user.conf
-      ln -s ${userConf}         $latest_folder/conf.d/99-user.conf
+      # local.conf (indirect priority 51)
+      ${optionalString (cfg.localConf != "") ''
+      ln -s ${localConf}        $support_folder/local.conf
+      ln -s ${localConf}        $latest_folder/local.conf
```

Comment:
Nice catch!
Indeed the latest `local.conf` wasn't used.

Even if both `local.conf` have the same contents, I amended the commit so latest `51-local.conf` to look for latest `local.conf`.  (https://github.com/NixOS/nixpkgs/pull/16730/commits/da342b0b017a44b36c536870ba4b75b18a2c2ebf#diff-376672cba12c87d4103e6134632e306fR164)

It feels cleaner and it will make things easier if in the future we want to have a different `local.conf` for each version.